client/extensions/auto-translator/src/widget/translation-dialog/form.tsx

```diff
@@ -212,6 +215,66 @@ const getTranslation = (payload: TranslationPayload) =>
     payload: { service: "translate", item: payload },
   });
 
+const isManualTranslationDirty = (
+  values: ReturnType<
```

Comment:
can use the follwing for better readability
```suggestion
  type FormikContextType = ReturnType<typeof useFormikContext<TranslationDialogFormProps>>;

const isManualTranslationDirty = (
  values: FormikContextType["values"],
  getFieldMeta: FormikContextType["getFieldMeta"]
```

Reply:
good catch